index.bs

```diff
@@ -1145,8 +1145,8 @@ run the following steps:
 1. If |name|'s [=string/length=] is 0:
     1. If |value| contains U+003D (=), then return failure.
     1. If |value|'s [=string/length=] is 0, then return failure.
-    1. If |value|, [=byte-lowercased=], [=byte sequence/starts with=] \``__host-`\`, \``__http-`\`, or \``__secure-`\`, then return failure. 
-1. If |name|, [=byte-lowercased=], [=byte sequence/starts with=] \``__http-`\`, then return failure.
+    1. If |value|, [=byte-lowercased=], [=byte sequence/starts with=] \``__host-`\`, \``__hosthttp-`\`, \``__http-`\`, or \``__secure-`\`, then return failure. 
+1. If |name|, [=byte-lowercased=], [=byte sequence/starts with=] \``__http-`\`, or \``__hosthttp-`\`, then return failure.
```

Comment:
There's no need for a comma before or here. It's only two options.

Reply:
Removed comma